ddtrace/internal/compat.py

```diff
@@ -184,7 +184,6 @@ def getrandbits(k):
         textwrap.dedent(
             """
     import functools
-    import asyncio
```

Comment:
@brettlangdon I think this code was introduced by you in https://github.com/DataDog/dd-trace-py/pull/751. Do you reckon we should replace this import with something else (e.g. checking that we can find the loader), or can we just do away with it?

Reply:
oh, this function used to do `@asyncio.coroutine`, which is does not anymore, so yes it can likely be removed. it was likely just missed before.